kombu/utils/scheduling.py

```diff
@@ -88,17 +97,17 @@ def rotate(self, last_used):
         return last_used
 
 
-class priority_cycle(round_robin_cycle):
+class priority_cycle(round_robin_cycle[ResourceType]):
     """Cycle that repeats items in order."""
 
-    def rotate(self, last_used):
+    def rotate(self, last_used: Any) -> None:
```

Comment:
anything but any https://sethmlarson.dev/blog/tests-arent-enough-case-study-after-adding-types-to-urllib3#anything-but-any